apps/reactjs/src/utils.tsx

```diff
@@ -4,8 +4,8 @@ import { debugServer } from "nexys/dist/utils";
 export const nexys: Nexys = new Nexys("d1ae511a-af59-46a8-8db7-d0db795be3c4", {
   appName: "test2",
   debug: true,
-  server: debugServer,
-  ignoreType: "INFO",
+  //server: debugServer,
+  //ignoreType: "INFO",
```

Comment:
yorum satırlarını yok edebilir misin

Reply:
belirtilen yerdeki yorum satırları reactjs paketine ait internal testler için kullanıyoruz. kullanıcılar zaten /apps kısmındaki uygulamalara erişemiyorlar docs-v2 dışında. bazen debugServer'ı aktif etmek gerekebiliyor.